yt/frontends/adaptahop/io.py

```diff
@@ -194,6 +185,10 @@ def members(self, ihalo):
             members = fpu.read_attrs(todo.pop(0))["particle_identities"]
         return members
 
+    def _sorted_chunk_iterator(self, chunks):
+        data_files = self._get_data_files(chunks)
+        yield from sorted(data_files, key=attrgetter("filename"))
```

Comment:
@matthewturk just pointing out an example where I added an override of `_sorted_chunk_iterator` so that the frontend continues only sorting by filename. The change in `ahf` is similar.